.gitignore

```diff
@@ -1,2 +1,3 @@
 .build/
```

Comment:
Can you please remove the changes in this file so we can consider only the changes in docker/dirigible folder?

Reply:
You are still including a change in `.gitignore` file. Go for: 

```
git rm --cached .gitignore && git commit -m "Remove .gitgnore" && git push
```